### ISSUE TITLE
missing .js extension

### ISSUE THREAD

**Issue by christophe-g** [material-components/material-components-web-components#926]
a lot of `.js` extension are not present in import statement (for instance `import {classMap} from 'lit-html/directives/class-map;` as opposed to `import {classMap} from 'lit-html/directives/class-map.js;`).

This (and the fact that typescript does not add the extension while compiling, see https://github.com/microsoft/TypeScript/issues/33882) creates problems when you use tools like [snowpack](https://www.snowpack.dev/) where exact name mapping is needed. 

Snowpack config in `package.json` requires smth like: 
```json
"snowpack": {
    "webDependencies": [
      "lit-element",
      "lit-html",
      "lit-html/directives/cache.js",
      "lit-html/directives/class-map.js"
 ]}
```
Import statements without `.js`  cause snowpack dependency tree to be incomplete. 

Will propose a PR shortly.

**Comment by asyncLiz** [comment]
Thanks for the issue! I'll bring it up with the team to determine how we want to proceed.

**Comment by asyncLiz** [comment]
Added #939 to track some automated tooling to do this

**Comment by dfreedm** [comment]
Fixed with #927
